doc/rtd_environment.yml

```diff
@@ -31,4 +31,4 @@ dependencies:
   # pip packages
   - pip
   - pip:
-    - -e ../
+    - ../
```

Comment:
this works in the [GH workflow](https://readthedocs.org/projects/weldx/builds/16706275/) and on [RTD](https://readthedocs.org/projects/weldx/builds/16706275/) 
I don't think it is necessary to keep the `-e` here, what do you think @marscher ?

Reply:
If there is anything which needs to refer to to source code directory via the Python module, this would break with this change. Apparently there is no such reference right now.
Actually I would have thought, that at least Sphinx would refer to the source code directory via relative file links. But since the pip installed source tree is equivalent, nothing should break.